### ISSUE TITLE
[PRO] new warning - "Passing 'call' command to redis as is; blind passthrough has been deprecated and will be removed in redis-namespace 2.0"

### ISSUE THREAD

**Issue by xxx** [rmosolgo/graphql-ruby#3322]
**Describe the bug**

After updating redis-namespace today, we get a ton of spew of the following warning:

```
Passing 'call' command to redis as is; blind passthrough has been deprecated and will be removed in
redis-namespace 2.0 (at /home/mpd/.rvm/gems/ruby-2.7.2/gems/graphql-pro-1.17.4/lib/graphql
/pro/subscriptions/redis_storage.rb:169:in `each_subscription_id_legacy')
```

I'm not sure what timeline that 2.0 release is on.

**Versions**

`graphql` version: 1.12.3 (pro version 1.17.4)
`rails` (or other framework): 6.0.3.4
redis-rb: 4.2.5
redis-namespace: 1.8.1

(leaving the rest out because I don't think it's needed for this one).

**Comment by rmosolgo** [comment]
Derp, thanks for reporting this. `redis.call(...)` is _how_ you call Redis from Lua, but I see I accidentally did it in Ruby too 😖 . Just needs to be replaced with a `.smembers` call -- I'll follow up here when I've released a fix!

**Comment by rmosolgo** [comment]
I just shipped graphql-pro 1.17.5 which cleans up that usage of `.call` -- let me know if you have any more trouble after updating!
